plugins/dell-dock/README.md

```diff
@@ -4,6 +4,14 @@ Dell USB-C Dock
 ### Dell System
 Unlike previous Dell USB-C devices, a Dell system is not needed for updating.
 
+GUID Generation
+---------------
+
+These devices use several modified USB DeviceInstanceId values, e.g.
+
+ * `USB\VID_1D50&PID_60C6&hub`
+ * `USB\VID_1D50&PID_60C6&hub&embedded`
+
```

Comment:
I think with this one it's well understood which ones get which.

```
USB Hub1: USB\VID_413C&PID_B06F&hub
Usb Hub2: USB\VID_413C&PID_B06E&hub
EC: USB\VID_413C&PID_B06E&hub&embedded
Update Level: USB\VID_413C&PID_B06E&hub&status
MST Hub: MST-panamera-vmm5331-259
Thunderbolt controller: TBT-00d4b070
```

Reply:
Thanks, that's better.